aws_lambda_builders/workflows/nodejs_npm_esbuild/esbuild.py

```diff
@@ -101,6 +101,10 @@ def run(self, args, cwd=None):
 
 NON_CONFIGURABLE_VALUES = {"bundle", "platform", "outdir"}
 
+# Ignore the values below. These are options that Lambda Builders accepts for
+# Node.js related workflows, but are not relevant to esbuild itself.
+ESBUILD_IGNORE_VALUES = {"use_npm_ci", "entry_points"}
```

Comment:
I explored other ways of ignoring these types of unrelated values too, however didn't find anything reliable (e.g. there's no API to use, and parsing the esbuild help text is very complicated. We can go with this solution for now as it fixes the immediate issue, and look for ways of making this more reliable moving forward.